config.json

```diff
@@ -1172,6 +1172,15 @@
         "Recursion",
         "Searching"
       ]
+    },
+    {
+      "slug": "two-fer",
+      "core": true,
+      "unlocked_by": "hello-world",
```

Comment:
If we use this as a core exercise, and I think we should, this (unlocked_by) should be `null`.